portfolio/src/main/webapp/script.js

```diff
@@ -11,18 +11,33 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
+ 
```

Comment:
Get rid of these empty spaces throughout the file.

Reply:
done